pkg/receiver/smartagentreceiver/output.go

```diff
@@ -195,6 +196,15 @@ func (out *output) Copy() types.Output {
 	return &cp
 }
 
+func (out *output) SendMetrics(metrics pmetric.Metrics) {
+	if out.nextMetricsConsumer == nil {
+		return
+	}
+	ctx := out.reporter.StartMetricsOp(context.Background())
+	err := out.nextMetricsConsumer.ConsumeMetrics(context.Background(), metrics)
```

Comment:
the monitor configs have an `extra_dimensions` field that should be applied to these

Reply:
I'll create an integration test that uses `extra_dimensions`, and use it to see if we can drive backwards compatibility, similar to what we have for histograms.